test/integration/mongodb-handshake/mongodb-handshake.prose.test.ts

```diff
@@ -110,6 +116,49 @@ describe('Handshake Prose Tests', function () {
     });
   }
 
+  context('Test 9: Valid container and FaaS provider', function () {
+    const env = [
+      ['AWS_EXECUTION_ENV', 'AWS_Lambda_java8'],
```

Comment:
```suggestion
      ['AWS_EXECUTION_ENV', 'AWS_Lambda_nodejs22.x'],
```

FFTR but FWIW this is what the value would be for our users

Reply:
This value is mandated by the spec test: https://github.com/mongodb/specifications/blob/master/source/mongodb-handshake/tests/README.md#:~:text=Valid%20container%20and%20FaaS%20provider.%20This%20test%20MUST%20verify%20that%20both%20the%20container%20metadata%20and%20the%20AWS%20Lambda%20metadata%20is%20present%20in%20client.env.

Reply:
I know but in practice that's never the value used for this driver which seems less productive to check our logic in a practical way. (But FFTR)